dbt_sugar/core/connectors/postgres_connector.py

```diff
@@ -37,3 +38,26 @@ def __init__(
             port=connection_params.get("port", str()),
         )
         self.engine = sqlalchemy.create_engine(self.connection_url)
+
+    def get_columns_from_table(
```

Comment:
Needed to add it because it was being call in doc, with the parameter so it was working for snowflake but not for postgresql.